src/screens/InputUsername/index.js

```diff
@@ -444,19 +445,15 @@ const styles = StyleSheet.create({
     width: '90%'
   },
   footer: {
-    paddingHorizontal: 20,
-    paddingVertical: 20,
+    position: 'absolute',
+    bottom: 0,
+    height: dimen.normalizeDimen(112),
+    width: '100%',
+    paddingHorizontal: dimen.normalizeDimen(20),
+    paddingBottom: dimen.normalizeDimen(20),
     backgroundColor: colors.white,
-    justifyContent: 'flex-end'
+    flexDirection: 'column',
+    justifyContent: 'space-between'
   },
-  textSmall: {
-    fontFamily: 'Inter',
-    fontStyle: 'normal',
-    fontWeight: 'normal',
-    fontSize: 10,
-    textAlign: 'center',
-    color: colors.emperor,
-    marginBottom: 20,
-    marginTop: 10
-  }
+  textSmallContainer: {flex: 1}
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The changes made here are good for improving the responsiveness of the application. By using `dimen.normalizeDimen`, the dimensions of the footer will be adjusted based on the screen size, which can help to ensure that the layout looks consistent across different devices. The change from `justifyContent: 'flex-end'` to `justifyContent: 'space-between'` in the footer style also improves the layout by distributing space evenly between the children along the vertical axis.


<!-- This is an auto-generated comment by OSS CodeRabbit -->